js/src/lib/Details/FileBrowser.js

```diff
@@ -28,8 +24,16 @@ export default class FileBrowser extends React.PureComponent {
 
   _fetchFiles() {
     this.setState({ error: null, files: null });
-    const url = this._getBaseURL() + '?meta';
+    const url = `https://data.jsdelivr.com/v1/package/npm/${this.props.objectID}@${this.props.version}`;
```

Comment:
Would prefer if this was pulled out into a separate function (eg. `getURLForPackageMetadata` or something like that)